doc/fitting/standard_fits.md

```diff
@@ -183,3 +183,22 @@ g(x) = \max(f(x), \text{background} - \text{height})
 ```{math}
 y = \min(g(x), \text{background})
 ```
+
+## Muon Momentum
+
+API Reference: ['MuonMomentum'](ibex_bluesky_core.fitting.MuonMomentum)
```

Comment:
This (backticks rather than quotes) makes it format like the others:

```suggestion
API Reference: [`MuonMomentum`](ibex_bluesky_core.fitting.MuonMomentum)
```